### ISSUE TITLE
CSRF_HEADER_NAME not respected.

### ISSUE THREAD

**Issue by geekashu** [marcgibbons/django-rest-swagger#599]
I have changed `CSRF_HEADER_NAME = 'HTTP_X_XSRF_TOKEN'` in my django settings file.

It seems this setting is not being respected, swagger uses default header `--header 'X-CSRFToken:` while making any API requests, this results in `CSRF Failed: CSRF token missing or incorrect.` Version - 2.1.0

**Comment by marcgibbons** [comment]
Yup - this is a bug. Need to pass along the token header name into the JS config.

Somewhere in here:
https://github.com/marcgibbons/django-rest-swagger/blob/master/rest_framework_swagger/static/rest_framework_swagger/init.js#L48

**Comment by arkadiybag** [comment]
add in your settings
`CSRF_HEADER_NAME = "CSRF_COOKIE"`

**Comment by cypreess** [comment]
+1
